bundles/ch.elexis.base.messages/src/ch/elexis/base/messages/MsgDetailDialog.java

```diff
@@ -97,9 +99,20 @@ public String getText(Object element) {
 				return anw.getLabel();
 			}
 		});
-		List<Anwender> users = getUsers();
-		cbTo.setInput(users);
-		cbTo.setSelection(new StructuredSelection(users.get(0)));
+	    List<Anwender> users = getUsers();
+	    cbTo.setInput(users);
+		String preferenceKey = getPreferenceKeyForUser();
+	    String savedRecipientId = ConfigServiceHolder.getUser(preferenceKey, null);
+	    if (savedRecipientId != null && !savedRecipientId.isEmpty()) {
+	        for (Anwender user : users) {
+	            if (user.getId().equals(savedRecipientId)) {
+	                cbTo.setSelection(new StructuredSelection(user));
+	                break;
+	            }
+	        }
+	    } else {
+			cbTo.setSelection(new StructuredSelection(users.get(0)));
+	    }
 
```

Comment:
Weshalb ist das eingerückt? Gibt es da ein Problem mit dem Formatter?

Reply:
Bei mir zeigt es richtig an wie es aussieht, wurde es nicht richtig in Git formatiert 
![grafik](https://github.com/elexis/elexis-3-base/assets/93644806/274a943f-3e7e-47a6-81f9-8cc95cdfe300)